game/hud/src/services/session/layout.ts

```diff
@@ -86,20 +92,26 @@ export function saySomething(s: string) : LayoutAction {
   }
 }
 
+export function adjustWidgetPositions() {
+  return {
+    type: ADJUST_POSITIONS,
+  }
+}
 
 
-let defaultWidgets = () => {
+const defaultWidgets = (): any => {
```

Comment:
8.5 specifically gives an example of a function defined using an arrow function as a // good